tests/testthat/test-utils.R

```diff
@@ -0,0 +1,17 @@
+# TODO: delete this test when we have any lints using xml_nodes_to_lint where the
+#   target node can span >1 line (e.g. an <expr>); until then, this test is here
+#   for coverage
+test_that("xml_nodes_to_lint handles >1-line nodes correctly", {
+  xml <- xml2::read_xml('<x line1="1" col1="1" line2="2" col2="10" start="8" end="25"></x>')
+  source_file = list(filename = "test", lines = c(`1` = "abcdefg", `2` = "hijklmnopq"))
```

Comment:
here's a lint

Reply:
thanks! is there a place to see these without checking out the Changes tab (which I don't always do)...

maybe I should write a precommit hook for this

Reply:
You can manually look at the action logs, e.g. [here](https://github.com/jimhester/lintr/runs/2093351910#step:9:14)
Other than that, locally running `devtools::load_all(); lint_package()` is probably the most convenient way.